src/main/java/com/chessix/sepa/pain/impl/Pain00100103.java

```diff
@@ -74,7 +78,7 @@ private void initializeDocument() {
 
 		// Payment Information
 		PaymentInstructionInformation3 pi = new PaymentInstructionInformation3();
-		pi.setPmtInfId(DocumentUtils.createUniqueId());
+		pi.setPmtInfId(paymentInfoId);
```

Comment:
I may have overlooked, but for downwards compatibility all callers to the constructor need to set paymentInfoId with DocumentsUtils.createUniqueId(). Is there a check when this is null?